pkg/host/utils.go

```diff
@@ -0,0 +1,18 @@
+package host
```

Comment:
Adding config to host breaks the motor build as viper is not compatible with WASM and Mobile

Reply:
good call, i rembered that and took out the helpers i wrote using some env vars but didn't remove the file entirely. ill do that.